commit 9c9c15f702d567ceed83826e66e1fa4a3c123eb1
Author: Narrat <autumn-wind@web.de>
Date:   Wed Jul 22 23:05:48 2020 +0200

    Customize the timer
    
    Start everything with the timer.
    Set type to oneshot.
    Default units won't work like they should.

diff --git a/PKGBUILD b/PKGBUILD
index 1b54dc9..d694c35 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -10,10 +10,18 @@ optdepends=('sudo: needed for recall function')
 replaces=('modprobed_db')
 conflicts=('modprobed_db')
 url="https://wiki.archlinux.org/index.php/Modprobed-db"
-source=("$pkgname-$pkgver.tar.gz::https://github.com/graysky2/$pkgname/archive/v$pkgver.tar.gz")
-sha256sums=('712fbc6ff40f439a688fc2e3d1fbd81226a24ef63ce37aa086c30cdd2c6e7047')
+source=("$pkgname-$pkgver.tar.gz::https://github.com/graysky2/$pkgname/archive/v$pkgver.tar.gz"
+        "adjust-units.patch")
+sha256sums=('712fbc6ff40f439a688fc2e3d1fbd81226a24ef63ce37aa086c30cdd2c6e7047'
+            '74a0c143f97fc5b9150935d1554a71f0ca04e6f1d87f8a0699d89544f24770f6')
 install=readme.install
 
+
+prepare() {
+	cd "$pkgname-$pkgver"
+	patch -Np1 -i $srcdir/adjust-units.patch
+}
+
 build() {
 	cd "$pkgname-$pkgver"
 	make
diff --git a/adjust-units.patch b/adjust-units.patch
new file mode 100644
index 0000000..592a4c0
--- /dev/null
+++ b/adjust-units.patch
@@ -0,0 +1,30 @@
+--- modprobed-db-2.44/init/modprobed-db.service
++++ modprobed-db-2.44/init/modprobed-db.service
+@@ -2,13 +2,8 @@
+ Description=modprobed-db scan and store new modules
+ Documentation=man:modprobed-db(8)
+ Documentation=https://wiki.archlinux.org/index.php/Modprobed-db
+-RequiresMountsFor=/home/
+-Wants=modprobed-db.timer
+ 
+ [Service]
+-Type=simple
++Type=oneshot
+ ExecStart=/usr/bin/modprobed-db storesilent
+ ExecStop=/usr/bin/modprobed-db storesilent
+-
+-[Install]
+-WantedBy=default.target
+
+--- modprobed-db-2.44/init/modprobed-db.timer
++++ modprobed-db-2.44/init/modprobed-db.timer
+@@ -3,4 +3,8 @@
+ PartOf=modprobed-db.service
+ 
+ [Timer]
++OnBootSec=1h
+ OnUnitActiveSec=6h
++
++[Install]
++WantedBy=timers.target
+
